### PR TITLE
Update version number, README, and Attribution.txt

### LINKED ISSUE #3: Update version number to 0.7.1 for Delhi dot release, README, and create Attribution.txt

### PR DESCRIPTION
**Pull request by cloudxxx8**
Update version number to 0.7.1 for Delhi dot release, README, and create Attribution.txt
fix #3

Signed-off-by: Cloud Tsai <cloudxxx8@gmail.com>

### CODE REVIEW

**Comment by tonyespy** [comment]
Actually, you included these in the new SDK Attribution.txt files, so I guess you should just merge that file with the file you've added here. Does that make sense?

**Comment by JPWKU** [comment]
@tonyespy @cloudxxx8 There is a fix in PR that @jamesrgregg provided that will solve this failure here https://github.com/edgexfoundry/device-modbus-go/pull/2

File: cmd/Attribution.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+The following open source projects are referenced by Modbus Device Service Go:
+
+goburrow/modbus (unspecified) - https://github.com/goburrow/modbus
+https://github.com/goburrow/modbus/blob/master/LICENSE
+
+tbrandon/mbserver (MIT) - https://github.com/tbrandon/mbserver
+https://github.com/tbrandon/mbserver/blob/master/LICENSE
```
Comment (anonymouse64) [inline cmd/Attribution.txt]: This library also imports https://github.com/goburrow/serial, so we need to include the license(s) for that project here as well.

Comment (anonymouse64) [reply cmd/Attribution.txt]: I just checked and that serial library doesn't include any other 3rd party deps in it, so you can just add the line for that project.

File: cmd/Attribution.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+BurntSushi/toml (MIT) - https://github.com/BurntSushi/toml
+https://github.com/BurntSushi/toml/blob/master/COPYING
+
+gorilla/mux 1.6.2 (BSD-3) - https://github.com/gorilla/mux
+https://github.com/gorilla/mux/blob/master/LICENSEE
```
Comment (anonymouse64) [inline cmd/Attribution.txt]: This is a typo, it should be `LICENSE` not `LICENSEE` (note the extra `E`)

File: cmd/Attribution.txt [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+hashicorp/consul 1.1.0 (Modzilla Public License 2.0) - https://github.com/hashicorp/consul
+https://github.com/hashicorp/consul/blob/master/LICENSE
```
Comment (anonymouse64) [inline cmd/Attribution.txt]: There's a ton of libraries that this pulls in, and we need to attribute all of them, unless you or someone else can prove that we don't use the code contained in them. Personally, I think it's easier to just attribute all of the dependencies than try to reason about consul's api library... Here's all the libraries I found using godoc.org's imports feature:

* https://github.com/hashicorp/go-cleanhttp
* https://github.com/hashicorp/go-rootcerts
* https://github.com/mitchellh/go-homedir
* https://github.com/mitchellh/mapstructure
* https://github.com/hashicorp/serf (not that we only end up importing the coordinate package here, so I didn't follow the dep chain for the other folders here, so just include the LICENSE for this specific repo)
* https://github.com/armon/go-metrics
* https://github.com/hashicorp/go-immutable-radix
* https://github.com/hashicorp/golang-lru

So just add a line for each library from above.

File: cmd/Attribution.txt [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+https://github.com/robfig/cron/blob/master/LICENSE
+
+mgo v2 (BSD-2) - https://github.com/go-mgo/mgo
+https://github.com/go-mgo/mgo/blob/v2-unstable/LICENSE
+
+mgo/bson v2 (unspecified) - https://github.com/go-mgo/mgo/tree/v2/bson
+https://github.com/go-mgo/mgo/blob/v2/bson/LICENSE
+
+gopkg.in/yaml v2 (Apache 2.0) - https://github.com/go-yaml/yaml/tree/v2
+https://github.com/go-yaml/yaml/blob/v2/LICENSE
```
Comment (anonymouse64) [inline cmd/Attribution.txt]: This file also needs to include a direct reference for `edgex-go` and `device-sdk-go` because it imports those packages.

**Pull request merged**